roles/common/tasks/main.yml

```diff
@@ -20,6 +20,26 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: set selinux to permissive on redhat
+  selinux:
+    policy: targeted
+    state: permissive
+  when: ansible_distribution == "RedHat"
+  tags: selinux
+
+- name: Register with RHN
+  redhat_subscription:
+    state: present
+    username: "{{ redhat.user }}"
+    password: "{{ redhat.pass }}"
+  when: ansible_distribution == "RedHat"
+  tags: rhn
+
```

Comment:
if you look at the following task it runs attach  ( don't need to specify the pool ).

https://github.com/blueboxgroup/ursula/pull/2383/files/4c234a0ea932cfca7dfcc9979b01f48b08f0c56f#diff-3cd6f6e3b6e4c2ea1695765d4b707c63R39